### PR TITLE
please check

### PR DESCRIPTION
**Pull request by robinburenhva**
appointments

### CODE REVIEW

File: src/pages/create/creation-framework.tsx [hunk 1]
```diff
@@ -40,7 +41,13 @@ const pages: (React.FC<FormProps> | FormPage)[] = [
         parts: [
             Personalia,
             Insurance,
-            Circumstances,
+            Circumstances
+        ]
+    },
+    {
+        title: 'APpointment',
```
Comment (ZIggyVeerman) [inline src/pages/create/creation-framework.tsx:48]: Typo: APpointment

File: src/pages/forms/appointment/query/get-appointment.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        extra {
+            description
+            date
+            time
+        }
+    }
+}
+`;
+
+export const useGetAppointments = () => useQuery(GET_APPOINTMENTS, {  });
```
Comment (ZIggyVeerman) [inline src/pages/forms/appointment/query/get-appointment.ts:26]: you should add the types back in

File: src/pages/forms/appointment/mutation/save-appointment.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    saveAppointments(
+        funeralId: $id,
+        appointments: $appointments
+    ){
+        dateConferenceContent
+    }
+}
+`;
+
+export const useSaveAppointments = () => useMutation(SAVE_APPOINTMENTS);
```
Comment (ZIggyVeerman) [inline src/pages/forms/appointment/mutation/save-appointment.ts:15]: you should add the types back in

**Review by ZIggyVeerman — CHANGES_REQUESTED** [review]
Changes should be made comments added, Also all checks fail for the circleCI maybe check what is happening there.

**Review by joeypothoff — APPROVED** [review]
Looks good to me.

**Review by ZIggyVeerman — APPROVED** [review]

**Pull request merged**